src/cov/Coverage.java

```diff
@@ -528,12 +537,17 @@ public static Member getMemberContaining(CoverageAtom cov) {
    * Increments the count for the given branch by 1.
    */
   public static void touch(Branch br) {
-    Class<?> cls = Reflection.classForName(br.getClassName());
-    if (br.branch) {
-      getTrueBranches(cls)[br.branchNumber]++;
-    } else {
-      getFalseBranches(cls)[br.branchNumber]++;
+    try {
+      Class<?> cls = Class.forName(br.getClassName());
+      if (br.branch) {
+        getTrueBranches(cls)[br.branchNumber]++;
+      } else {
+        getFalseBranches(cls)[br.branchNumber]++;
+      }
+    } catch (ClassNotFoundException e) {
```

Comment:
Please narrow the scope of every try...catch block to be as small as possible -- in this case, to just be the Class.forName call.  That makes its purpose more clear.